library/core/src/lib.rs

```diff
@@ -49,6 +49,10 @@
 //
 // This cfg won't affect doc tests.
 #![cfg(not(test))]
+// To run libcore tests without x.py without ending up with two copies of libcore, Miri needs to be
+// able to "empty" this crate. See <https://github.com/rust-lang/miri-test-libstd/issues/4>.
+// rustc itself never sets the feature, so this line has no affect there.
+#![cfg(any(not(feature = "miri-test-libstd"), doctest))]
```

Comment:
@hyd-dev shouldn't this also contain `, test`?

Reply:
Ah, I guess it doesn't since the crate is anyway empty then due to the `cfg` above.

Reply:
If `core` gets unit tests one day, it's possible to forget to add `test` here when `#[cfg(not(test))]` is removed, though, so maybe it's still better to add `test` here.

Reply:
Yeah probably better to be consistent here.